vllm/model_executor/models/commandr.py

```diff
@@ -292,7 +294,8 @@ def forward(
         return hidden_states
 
 
-class CohereForCausalLM(nn.Module):
+class CohereForCausalLM(nn.Module, SupportsLoRA):
+    supports_lora = True
```

Comment:
Actually, there is no need to set `supports_lora=True` since it's automatically set via inheritance.

Reply:
You are right. I did not add this initially since in your [initial pull request](https://github.com/DarkLight1337/vllm-rocm/blob/6a75147b5f4c7c6e08b6c7e655d410fa3750d0f2/vllm/model_executor/models/interfaces.py#L55) you did not specify default value for that. I will correct this